examples/transaction_lifecycle_example/main.py

```
@@ -0,0 +1,179 @@
+    
+    try:
+        # Step 1: Submit destruction transaction
+        tx_hash = client.destroy_stream(stream_id)
+        print(f"   Destruction submitted: {tx_hash}")
+        
+        # Step 2: Wait for confirmation
+        print("   ⏳ Waiting for destruction to be mined...")
+        client.wait_for_tx(tx_hash)
+        print("✅ Stream destroyed and confirmed on-chain")
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)